workspaces/sandbox/plugins/sandbox/src/components/SandboxCatalog/SandboxCatalogBanner.tsx

```diff
@@ -22,25 +22,21 @@ import Skeleton from '@mui/material/Skeleton';
 import { useTheme } from '@mui/material/styles';
 import Image from '../../assets/images/sandbox-banner-image.svg';
 import { useSandboxContext } from '../../hooks/useSandboxContext';
-
-const TRIAL_DURATION_DAYS = 30;
+import { calculateDaysBetweenDates } from '../common';
```

Comment:
```suggestion
import { calculateDaysBetweenDates } from '../../utils/common';
```

Reply:
Its a relative path issue

Reply:
thanks!

Reply:
It's weird they were passing on my machine 😕